File: skills/quality/dry-test/fixtures/java/Clone.java
class Clone {
    String alpha(int value) {
        // Only alpha carries a comment. Comments are spelling, so beta must still match it.
        String label = "alpha: ";
        int total = value + 1;
        if (total > 10) {
            return label + Integer.toString(total);
        }
        return label;
    }

    String beta(int number) {
        String tag = "beta: ";
        int amount = number + 2;
        if (amount > 20) {
            return tag + Integer.toString(amount);
        }
        return tag;
    }
}

File: skills/quality/dry-test/fixtures/kotlin/clone.kt
fun alpha(value: Int): String {
    // Only alpha carries a comment. Comments are spelling, so beta must still match it.
    val label = "alpha: "
    val total = value + 1
    if (total > 10) {
        return label + total.toString()
    }
    return label
}

fun beta(number: Int): String {
    val tag = "beta: "
    val amount = number + 2
    if (amount > 20) {
        return tag + amount.toString()
    }
    return tag
}
